src/shared/components/Layout/Layout.styled.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, styled } from "@mui/material";
+import { headerHeight } from "./Header/Header.styled.tsx";
+
+export const LayoutContainer = styled(Box)`
+  display: flex;
+  height: 100vh;
+  width: 100vw;
```

Comment:
I suggest to use `vw` and `vh` only if you absolutely have to. Can you use `100%` instead?

Reply:
For `height: 100%` to work, you have to give it to `<body>` and `<html>` too